README.md

```diff
@@ -130,6 +130,12 @@ config :ibanity, :timeout, [
 ]
 ```
 
+In your test configuration :
+
+```elixir
+config :ibanity, test: true
+```
+
```

Comment:
Could you move this under `## Test` in line 137 please?

Reply:
This is no longer necessary with your suggestion